### PR TITLE
chore: Update Readme.md

### PR DESCRIPTION
**Pull request by greg-gol**
@marcinaylien, can we create a request for stories without using query object? (just as I presented in the readme)
I want to avoid unnecessary complexity into challange.

### CODE REVIEW

**Comment by greg-gol** [comment]
Feel free to let me know if you'd rephrase anything or add / simplify it.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
 ### Description
 For this project, you will provide a search bar that allows users to search via entities using the `/autocomplete` endpoint and display related stories using the `/stories` endpoint.
 
-We :heart: React and we already created a boilerplate project using [Create React App](https://create-react-app.dev/). However, you are free to use any frameworks and libraries you choose. The prototype is just a suggestion — you have complete creative freedom!
+We :heart: React and we already created a boilerplate project using [Create React App](https://create-react-app.dev/). Feel free to add any extra depencency that normally would be necessary or would make an engineers life easier. The prototype is just a suggestion — you have complete creative freedom!
```
Comment (marcinaylien) [inline README.md:15]: `would make an engineer's life easier` or `would make life of an engineer easier`

File: README.md [hunk 2]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
...
+
+## Required problems to resolve:
+  1. Webpack configuration that will deal with CORS issue while calling news-api.
+  2. Deliver code meeting acceptance criteria from the challange.
+  3. Make the design responsive so it'll render properly also on mobile devices.
+  4. Deliver unit tests covering critical logic of the project.
+  5. Bonus: Deliver e2e tests using Cypress.
+
+### AC:
+  * Application home page contains a search bar that let's a user type text in it (look at the simple design below).
```
Comment (marcinaylien) [inline README.md:26]: `let's a user type text` > `lets user type text` or `allows user to type text`

File: README.md [hunk 3]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
...
+  * Once text is provided into search bar, it'll fetch autocomplete results and present them in a selectable dropdown.
```
Comment (marcinaylien) [inline README.md:27]: consistent tense -> let's use present simple

File: README.md [hunk 4]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
...
+  * Selected entity from a dropdown replace provided text in the search bar.
```
Comment (marcinaylien) [inline README.md:28]: replace -> replaces

File: README.md [hunk 5]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
...
+  * Clicking search button fetch stories using news-api with provided entity in the search bar.
```
Comment (marcinaylien) [inline README.md:29]: fetch -> fetches

File: README.md [hunk 6]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
...
+  * Fetched data is presented as a list of stories below the search bar (look at the simple desing below).
+  * Whole application should run without any errors following happy.
```
Comment (marcinaylien) [inline README.md:31]: following happy?

File: README.md [hunk 7]
```diff
@@ -12,7 +12,24 @@ We need you to develop a Client (UI) architecture that allows users to search am
 ### Description
 For this project, you will provide a search bar that allows users to search via entities using the `/autocomplete` endpoint and display related stories using the `/stories` endpoint.
 
-We :heart: React and we already created a boilerplate project using [Create React App](https://create-react-app.dev/). However, you are free to use any frameworks and libraries you choose. The prototype is just a suggestion — you have complete creative freedom!
+We :heart: React and we already created a boilerplate project using [Create React App](https://create-react-app.dev/). Feel free to add any extra depencency that normally would be necessary or would make an engineers life easier. The prototype is just a suggestion — you have complete creative freedom!
+
+
+## Required problems to resolve:
```
Comment (marcinaylien) [inline README.md:18]: Is 4 hours enough to do it all?

File: README.md (hunk 6 above)
Comment (greg-gol) [reply README.md:31]: flow :D

File: README.md (hunk 7 above)
Comment (greg-gol) [reply README.md:18]: After conversation on slack we concluded we'll make part of the requirements regarding autocomplete optional.

**Review by marcinaylien — APPROVED** [review]

**Pull request merged**